zhaquirks/xiaomi/aqara/switch_h1.py

```diff
@@ -0,0 +1,158 @@
+                    Scenes.cluster_id,  # 5
+                    OnOff.cluster_id,  # 6
+                    Alarms.cluster_id,  # 9
+                    XiaomiMeteringCluster.cluster_id,  # 0x0702
+                    0x0B04,
+                ],
+                OUTPUT_CLUSTERS: [
+                    Time.cluster_id,  # 0x000a
+                    Ota.cluster_id,  # 0x0019
+                ],
+            },
+            242: {
+                PROFILE_ID: 41440,
+                DEVICE_TYPE: 0x0061,
+                INPUT_CLUSTERS: [],
+                OUTPUT_CLUSTERS: [
+                    GreenPowerProxy.cluster_id,  # 0x0021
+                ],
+            },
+        },
+    }
+
+    replacement = {
+        ENDPOINTS: {
+            1: {
+                DEVICE_TYPE: zha.DeviceType.ON_OFF_SWITCH,
+                INPUT_CLUSTERS: [
+                    BasicCluster,  # 0
+                    DeviceTemperatureCluster.cluster_id,  # 2
+                    Identify.cluster_id,  # 3
+                    Groups.cluster_id,  # 4
+                    Scenes.cluster_id,  # 5
+                    OnOffCluster,  # 6
+                    Alarms.cluster_id,  # 9
+                    MultistateInputCluster,  # 18
+                    XiaomiMeteringCluster.cluster_id,  # 0x0702
+                    OppleSwitchCluster,  # 0xFCC0 / 64704
+                    0x0B04,
+                ],
+                OUTPUT_CLUSTERS: [
+                    Time.cluster_id,
+                    Ota.cluster_id,
+                ],
+            },
+            41: {
+                PROFILE_ID: zha.PROFILE_ID,
+                DEVICE_TYPE: zha.DeviceType.ON_OFF_SWITCH,
+                INPUT_CLUSTERS: [
+                    MultistateInputCluster,  # 18
+                ],
+                OUTPUT_CLUSTERS: [],
+            },
+            242: {
+                PROFILE_ID: 41440,
+                DEVICE_TYPE: 97,
+                INPUT_CLUSTERS: [],
+                OUTPUT_CLUSTERS: [GreenPowerProxy.cluster_id],
+            },
+        },
+    }
+
+    device_automation_triggers = {
+        (SHORT_PRESS, BUTTON): {
+            ENDPOINT_ID: 41,
+            CLUSTER_ID: 18,
+            COMMAND: f"41_{COMMAND_SINGLE}",
+            ARGS: {ATTR_ID: 0x0055, PRESS_TYPE: COMMAND_SINGLE, VALUE: 1},
+        },
+        (DOUBLE_PRESS, BUTTON): {
+            ENDPOINT_ID: 41,
+            CLUSTER_ID: 18,
+            COMMAND: f"41_{COMMAND_DOUBLE}",
+            ARGS: {ATTR_ID: 0x0055, PRESS_TYPE: COMMAND_DOUBLE, VALUE: 2},
+        },
+        # when triple-pressed, you're technically getting an attribute update call, so not sure if it can be used
+        (TRIPLE_PRESS, BUTTON): {
+            ENDPOINT_ID: 1,
+            CLUSTER_ID: 0,
+            COMMAND: COMMAND_ATTRIBUTE_UPDATED,
+            ARGS: {
+                ATTRIBUTE_ID: 5,
+                ATTRIBUTE_NAME: "model",
+                VALUE: "lumi.switch.n1aeu1",
+            },
+        },
```

Comment:
This is interesting.
Normally, the Aqara sensors send an attribute report with their model when the "reset button" is pressed once (which doesn't actually reset the device).

But I guess if -- for this device -- it's only sent when tripple-pressing the button, it should be fine.

Reply:
... although I do wonder if this can cause a "tripple press" event to be fired when:
(1) the button is first paired
(2) the button is re-paired (probably doesn't re-read the attribute again though)
(3) the model attribute is read through the clusters UI. <- Could you check that?

Reply:
(2) and (3) do indeed cause "triple press" events to be fired so I removed this trigger altogether since it may cause some unexpected behaviour for people who may want to use it.

I also noticed the following comment in the general cluster implementation which further strengthens my doubts for having "triple press" events for these switches: https://github.com/zigpy/zha-device-handlers/blob/dev/zhaquirks/xiaomi/__init__.py#L236-L239